examples/api/lib/widgets/overscroll_indicator/glowing_overscroll_indicator.0.dart

```diff
@@ -22,30 +25,71 @@ class GlowingOverscrollIndicatorExampleApp extends StatelessWidget {
   }
 }
 
+const Set<PointerDeviceKind> allPointers = <PointerDeviceKind>{...PointerDeviceKind.values};
+
+/// Passing this class into [MaterialApp.scrollBehavior] ensures that a
+/// [GlowingOverscrollIndicator] is created, regardless of the target platform.
```

Comment:
```suggestion
// Passing this class into [MaterialApp.scrollBehavior] ensures that a
// [GlowingOverscrollIndicator] is created, regardless of the target platform.
```

`///` is required for public API. this is just part of a sample.